src/components/notification/notification.vue

```diff
@@ -1,18 +1,35 @@
 <template>
-  <component :is="clickable ? 'button' : 'div'" :class="`apos-notification ${modifier}`" role="alert">
+  <component 
+    :is="clickable ? 'button' : 'div'" 
+    :class="modifierClass" 
+    :role="clickable ? false : 'alert'"
```

Comment:
Why would it not be an alert if it's clickable?

Reply:
a11y complains

Reply:
Oh, because it's on a button element. I think this should still be an alert. The button element should be inside the alerter.